app/styles/_library/components/_whats-on.scss

```diff
@@ -0,0 +1,350 @@
+
+      @include media(">=medium") {
+        grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 3;
+        grid-row-end: 4;
+        width: 182px;
+      }
+
+      svg {
+        max-height: 54px;
+        max-width: 54px;
+      }
+
+      .play-button-text {
+        color: rgb(var(--color-background));
+        font-family: Open Sans;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 16px;
+        letter-spacing: 1.6px;
+        text-transform: capitalize;
+      }
+
+      .play-button-text:after {
+        content: 'listen';
+        text-transform: capitalize;
+
+        @include media(">=medium") {
+          content: 'listen live';
+        }
+      }
+    }
+
+    .whats-on-show-title {
+      @include typeface(header, 10);
+
+      color: rgb(var(--color-primary-2));
+      font-weight: normal;
+      margin-bottom: 8px;
+      text-align: left;
+
+      @include media(">=medium") {
+        font-size: 52px;
+        grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        line-height: 60px;
+        margin-bottom: 16px;
+      }
+    }
+
+    .whats-on-show-description {
+      color: rgb(var(--color-text));
+      font-family: Open Sans;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 14.22px;
+      margin-bottom: 24px;
+      text-align: left;
+
+      @include media(">=medium") {
+        font-size: 18px;
+        grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 2;
+        grid-row-end: 3;
+        line-height: 25px;
+      }
+    }
+  }
+
+  .whats-on-episode {
+    margin-bottom: 8px;
+    padding-left: 20px;
+    padding-right: 20px;
+    width: 100%;
+
+    .whats-on-episode-segments,
+    .whats-on-episode-description {
+      background-image: linear-gradient(#EAEFF0,#EAEFF0);
+      background-repeat: no-repeat;
+    }
+
+    .whats-on-episode-segments .whats-on-episode-segment .whats-on-episode-segment-text,
+    .whats-on-episode-description .whats-on-episode-description-text {
+      color: rgb(var(--color-text));;
+      font-family: Open Sans;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: normal;
+      line-height: 16px;
+
+      @include media(">=medium") {
+        font-size: 18px;
+        line-height: 25px;
+      }
+    }
+
+    .whats-on-episode-segments {
+      align-items: center;
+      background-position: 4px top;
+      background-size: 2px 100%;
+      display: flex;
+      flex-direction: column;
+      padding-bottom: 10px;
+      padding-top: 10px;
+
+      @include media(">=medium") {
+        background-position: 7px top;
+      }
+
+      .whats-on-episode-segment {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        padding-bottom: 10px;
+        padding-top: 10px;
+        width: 100%;
+
+        .whats-on-episode-segment-dot {
+          background-color: #EAEFF0;
```

Comment:
It's a bit weird that the designs aren't using one of our white label colors here. Really would like to use tokens and not hard-coded colors especially in the base white label styles. I understand that's this is what is in the designs, maybe we can talk to design about this.